app/models/repository.rb

```diff
@@ -21,4 +21,77 @@ def self.clear_all
   def url
     "https://github.com/" + owner + "/" + name
   end
+
+  def release_notes_new_features
+    release_notes = latest_release_notes.downcase
+    release_notes_from_header = release_notes_features_header release_notes
+    unless release_notes_from_header.nil?
```

Comment:
`if release_notes_from_header` - and if you want it can be done as an `if` modifier (but this one is up to you - depends on the style you like):
```
return release_notes_from_header if release_notes_from_header
```